airgap/uds-dev-stack/zarf.yaml

```diff
@@ -0,0 +1,70 @@
+
+components:
+  - name: uds-dev-stack-airgap-images
+    files:
+      - source: local-path-provisioner.tar
+        target: local-path-provisioner.tar
+      - source: busybox.tar
+        target: busybox.tar
+      - source: wolfi-base.tar
+        target: wolfi-base.tar
+      - source: pause.tar
+        target: pause.tar
+      - source: nginx.tar
+        target: nginx.tar
+      - source: metallb-controller.tar
+        target: metallb-controller.tar
+      - source: metallb-speaker.tar
+        target: metallb-speaker.tar
+      - source: frr.tar
+        target: frr.tar
+      - source: minio.tar
+        target: minio.tar
+      - source: minio-mc.tar
+        target: minio-mc.tar
+    actions:
+      onCreate:
+        before:
+          # renovate: datasource=docker depName=rancher/local-path-provisioner
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" rancher/local-path-provisioner:v0.0.31 local-path-provisioner.tar
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" cgr.dev/chainguard/busybox:latest busybox.tar
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" cgr.dev/chainguard/wolfi-base:latest wolfi-base.tar
+          # renovate: datasource=docker depName=registry.k8s.io/pause
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" registry.k8s.io/pause:3.10 pause.tar
+          # renovate: datasource=docker depName=quay.io/nginx/nginx-unprivileged versioning=docker
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/nginx/nginx-unprivileged:1.28.0-alpine nginx.tar
+          # renovate: datasource=docker depName=quay.io/metallb/controller
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/metallb/controller:v0.14.9 metallb-controller.tar
+          # renovate: datasource=docker depName=quay.io/metallb/speaker
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/metallb/speaker:v0.14.9 metallb-speaker.tar
+          # renovate: datasource=docker depName=quay.io/frrouting/frr
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/frrouting/frr:9.1.0 frr.tar
+          # renovate: datasource=docker depName=quay.io/minio/minio versioning=regex:^RELEASE\.(?<major>\d+)-(?<minor>\d+)-(?<patch>\d+)T(?<build>\d+)-(?<revision>\d+)-\d+Z$
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/minio/minio:RELEASE.2025-04-22T22-12-26Z minio.tar
+          # renovate: datasource=docker depName=quay.io/minio/mc versioning=regex:^RELEASE\.(?<major>\d+)-(?<minor>\d+)-(?<patch>\d+)T(?<build>\d+)-(?<revision>\d+)-\d+Z$
+          - cmd: ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" quay.io/minio/mc:RELEASE.2024-11-21T17-21-54Z minio-mc.tar
+            description: Pull the airgap images for uds-dev-stack
```

Comment:
small nit: this description is only set on the last command here, but really should cover all of these I think? We could just using a single multi-line command block for all, or add descriptions to each.

Reply:
I actually wanted to do multiple commands because multi-line block can make it hard to know what image pull failed if it does. Will add descriptions to all